### DISCUSSION TITLE
What information is sent from the user to the engine?

### DISCUSSION THREAD

**Discussion by mehr32** [searxng/searxng#2675] [Q&A] [answered]
Hello . I want to know what information about me is sent to search engines using searxng?

**Comment by return42** [comment]
You need our help, we need your help / my third and last call:

Can you please help to translate SearXNG's UI:

- Persian: https://translate.codeberg.org/translate/searxng/searxng/fa/?q=state:%3Ctranslated

Consider it an educational measure that I will not answer any more questions from you unless you are also willing to give something back to the SearXNG community ..

**Reply by mehr32** [reply]
ommm. This was a question someone else asked me. why not . I will definitely help translate it!

**Reply by return42** [reply]
> ommm. This was a question someone else asked me

It was me, I asked you three times ..

> I will definitely help translate it!

don't hesitate .. go on .. I'm awaiting your suggestions.

**Reply by mehr32** [reply]
No no ! They asked me this privacy question, I opened this conversation on their behalf.

Eye at the first opportunity

**Comment by mehr32** [comment]
OK ! I finished the Farsi translation and got the bugs. It is up to you to confirm. Can you see my answer now? :)

**Reply by return42** [reply]
> I finished the Farsi translation and got the bugs. 

Wow .. thanks a lot :+1:

**Comment by return42** [comment, marked as answer]
Apart from the search term and other parameters such as language and time range you have selected, nothing is sent to the search engines by you or your browser.
